### PR TITLE
Added POSTGRES_PASSWORD and POSTGRES_DATABASE_NAME variables

### PR DESCRIPTION
**Pull request by pawel-yesware**
Added POSTGRES_PASSWORD and POSTGRES_DATABASE_NAME so we can specify our own passwords/databases instead of the generated ones.

### CODE REVIEW

File: functions [hunk 1]
```diff
@@ -270,8 +270,15 @@ service_url() {
   local SERVICE="$1"
   local SERVICE_ROOT="$PLUGIN_DATA_ROOT/$SERVICE"
 
+ 
   local PASSWORD="$(cat "$SERVICE_ROOT/PASSWORD")"
-  local DATABASE_NAME="$(get_database_name "$SERVICE")"
+
+  if [[ -e "$SERVICE_ROOT/DATABASE_NAME" ]];then
+    local DATABASE_NAME=$(cat "$SERVICE_ROOT/DATABASE_NAME")
```
Comment (josegonzalez) [inline functions]: We should create this file where necessary.

**Comment by josegonzalez** [comment]
I'd much rather see this go towards something like [this](https://github.com/dokku/dokku-postgres/pull/44#issuecomment-161105507) if at all possible.

**Comment by pawel-yesware** [comment]
I'm not quite sure. This would complicate things more than they should be. We wanted to be able to setup postgresql with custom password and database name during the creation process. I think Going the way you mentioned it will be too much hustle.

**Comment by josegonzalez** [comment]
You mean more work for yourself :)

Honestly I'm not in favor of merging if we are going down the route I suggested since then we'd need to maintain two interfaces for the same thing. @Flink thoughts? Either way, I defer to you.

**Comment by Flink** [comment]
@josegonzalez It seems that all these use cases we see can be addressed by your solution so I think this is what we’ll do :) I just didn’t take the time recently to code something :grimacing:

**Comment by josegonzalez** [comment]
I'm going to close this PR as it is unlikely to be merged as is and the ui it exposes is not the end goal.

**Pull request closed without merging**